viewer/src/public/types.ts

```diff
@@ -7,31 +7,31 @@ import { SectorGeometry } from '@/datamodels/cad/sector/types';
 import { SectorQuads } from '@/datamodels/cad/rendering/types';
 import { SectorCuller } from '@/internal';
 
-export type RevealOptions = {
+export interface RevealOptions {
```

Comment:
There is some difference between interface and type alias for typedoc and how to document it. Type declaration is more compact, but you can't document props without jsdoc. That how it looks:

![image](https://user-images.githubusercontent.com/4989157/88167147-a8b5f580-cc18-11ea-9bc5-05eb51836579.png)

But with interface keyword and prop-based docs it looks like that:

![image](https://user-images.githubusercontent.com/4989157/88167490-3396f000-cc19-11ea-8f2e-54014060def7.png)

It's much more verbose for interface declaration. But you don't have to write a separate jsdoc to document props. Not really sure which approach should be preferred. I think output for type alias is much easier to read, but interfaces are easier to document.

Reply:
I decided to stick with type + jsdoc here.